cmd/package.go

```diff
@@ -523,8 +523,20 @@ func DescribePackage(repo *util.Repo, packageName string) error {
 
 	var pkg *core.Package
 	var cmdConf *core.CmdConfig
+	var readme string
+
+	var tarReader *tar.Reader
+
+	// Load package (tar.gz or tar supported).
```

Comment:
I am talking about this region [here](https://github.com/mikelangelo-project/capstan/blob/master/cmd/package.go#L411-L419).